lib/model.js

```diff
@@ -459,7 +459,7 @@ assign(derived, {
 
       Object.keys(CONNECTION_STRING_OPTIONS).forEach((item) => {
         if (typeof this[item] !== 'undefined' && !req.query[item]) {
-          if (item === 'compression') {
+          if (item === 'compression' && this.compression) {
```

Comment:
so the idea is that it does not parse when `compression` is undefined, yea?

Reply:
exactly!